canonical/canonical.com
# Update /become-a-partner

## Done

- Updated the [/become-a-partner](https://canonical-com-887.demos.haus/partners/become-a-partner) page according to [the design](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?type=design&node-id=9427-18076&t=2gCmy0Ck7MrzwKzA-0)
- Removed outdated `is-paper` class
- Formatted `styles.scss`

## QA

- Compare [/become-a-partner](https://canonical-com-887.demos.haus/partners/become-a-partner) against [the design](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?type=design&node-id=9427-18076&t=2gCmy0Ck7MrzwKzA-0)
- Make sure that it looks correct and nothing is broken

## Issue / Card

Fixes WD-3745

## Before merging
If the image is the final one we want to use we should add it to the asset manager instead of hotlinking.
